frontend/src/components/CategoryForm/CategoryForm.tsx

```diff
@@ -0,0 +1,20 @@
+import { SyntheticEvent, useState } from 'react';
+
+const CategoryForm = ({ onSubmit }: { onSubmit: (value: string) => void }) => {
+  const [value, setValue] = useState('');
+
+  const submitForm = (e: SyntheticEvent) => {
+    e.preventDefault();
+    onSubmit(value);
+    setValue('');
+  };
+
+  return (
+    <form onSubmit={submitForm}>
```

Comment:
I see no reason to use form. A simple button with an onclick event would be just fine

Reply:
removed form